samples/cellular/smp_svr/sysbuild/mcuboot/prj.conf

```diff
@@ -0,0 +1,37 @@
+CONFIG_MAIN_STACK_SIZE=10240
+CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
+
+CONFIG_BOOT_SWAP_SAVE_ENCTLV=n
+CONFIG_BOOT_ENCRYPT_IMAGE=n
+
+CONFIG_BOOT_UPGRADE_ONLY=n
+CONFIG_BOOT_BOOTSTRAP=n
+
+### mbedTLS has its own heap
```

Comment:
```suggestion
### Mbed TLS has its own heap
```

Reply:
this config file was just copied from another project, so all of your nits are present in multiple (around 20) other config files.
if you'd like the changes made, perhaps a separate PR to fix them all?

Reply:
Right, in a separate PR then.